setuptools/command/bdist_egg.py

```diff
@@ -238,13 +239,26 @@ def run(self):
 
     def zap_pyfiles(self):
         log.info("Removing .py files from temporary directory")
+        py3 = sys.version_info >= (3, 0)
```

Comment:
Perhaps it's possible to invoke this behavior unconditionally (on Python 2 and Python 3). Presumably, it will be a null operation when no matching files are found.